pfp/index.html

```diff
@@ -93,12 +179,6 @@
             }
 
 
-            /* Page break after each new box */
-            .newBox {
-                page-break-before: always;
```

Comment:
It looks like I just removed instead of replaced the page break, but I did replace it with a break-before (updated version of page-break-before) on .newBoxBefore1 (line 72)